Build failed

When I am building using ./make-libpico.sh it has some errors at the end:

[100%] Linking CXX static library libpico.a
ar: stdio.c.obj: not found in archive
ar: stdio_usb.c.obj: not found in archive
ar: stdio_usb_descriptors.c.obj: not found in archive
make[2]: *** [libpico.a] Error 1
make[2]: *** Deleting file `libpico.a'
make[1]: *** [CMakeFiles/pico.dir/all] Error 2
make: *** [all] Error 2

I am on an Apple M2 on Mac OS Sequoia 15.1.1

Thank you for the help!

I just followed the instructions in this repo, cloning it, updating the submodules etc first, running python3 get.py and then running the script.

This is the log, please let me know if you prefer a different formating. 

```
krishnavenkatraman@Krishnas-MacBook-Air libpico % ./make-libpico.sh 
++ cd ../../pico-sdk/
++ pwd
+ export PICO_SDK_PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk
+ PICO_SDK_PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk
++ cd ../../system/arm-none-eabi/bin
++ pwd
+ export PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin:/opt/homebrew/opt/openjdk@17/bin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.4/shims:/Users/krishnavenkatraman/.nvm/versions/node/v21.5.0/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/nix/var/nix/profiles/default/bin/
+ PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin:/opt/homebrew/opt/openjdk@17/bin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.4/shims:/Users/krishnavenkatraman/.nvm/versions/node/v21.5.0/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/nix/var/nix/profiles/default/bin/
++ cd ../../system/riscv32-unknown-elf/bin
++ pwd
+ export PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/system/riscv32-unknown-elf/bin:/Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin:/opt/homebrew/opt/openjdk@17/bin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.4/shims:/Users/krishnavenkatraman/.nvm/versions/node/v21.5.0/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/nix/var/nix/profiles/default/bin/
+ PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/system/riscv32-unknown-elf/bin:/Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin:/opt/homebrew/opt/openjdk@17/bin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.4/shims:/Users/krishnavenkatraman/.nvm/versions/node/v21.5.0/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/nix/var/nix/profiles/default/bin/
++ cd ../../system/picotool
++ pwd
+ export PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/system/picotool:/Users/krishnavenkatraman/development/robots/arduino-pico/system/riscv32-unknown-elf/bin:/Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin:/opt/homebrew/opt/openjdk@17/bin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.4/shims:/Users/krishnavenkatraman/.nvm/versions/node/v21.5.0/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/nix/var/nix/profiles/default/bin/
+ PATH=/Users/krishnavenkatraman/development/robots/arduino-pico/system/picotool:/Users/krishnavenkatraman/development/robots/arduino-pico/system/riscv32-unknown-elf/bin:/Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin:/opt/homebrew/opt/openjdk@17/bin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.4/shims:/Users/krishnavenkatraman/.nvm/versions/node/v21.5.0/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.local/bin:/Users/krishnavenkatraman/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/nix/var/nix/profiles/default/bin/
+ rm -rf build-rp2040
+ mkdir build-rp2040
+ cd build-rp2040
+ CPU=rp2040
+ cmake ..
Building for CPU rp2040
Using PICO_SDK_PATH from environment ('/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk')
PICO_SDK_PATH is /Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk
Target board (PICO_BOARD) is 'pico_w'.
Using board configuration from /Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/boards/include/boards/pico_w.h
Pico Platform (PICO_PLATFORM) is 'rp2040'.
-- Defaulting build type to 'Release' since not specified.
Defaulting compiler (PICO_COMPILER) to 'pico_arm_cortex_m0plus_gcc' since not specified.
Configuring toolchain based on PICO_COMPILER 'pico_arm_cortex_m0plus_gcc'
Defaulting PICO_GCC_TRIPLE to 'arm-none-eabi'
-- The C compiler identification is GNU 14.2.0
-- The CXX compiler identification is GNU 14.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin/arm-none-eabi-gcc
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin/arm-none-eabi-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Users/krishnavenkatraman/development/robots/arduino-pico/system/arm-none-eabi/bin/arm-none-eabi-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Build type is RelWithDebInfo
-- Found Python3: /opt/homebrew/Frameworks/Python.framework/Versions/3.13/bin/python3.13 (found version "3.13.0") found components: Interpreter 
TinyUSB available at /Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040; enabling build support for USB.
BTstack available at /Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/btstack
cyw43-driver available at /Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/cyw43-driver
Pico W Bluetooth build support available.
lwIP available at /Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/lwip
Pico W Wi-Fi build support available.
mbedtls available at /Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/mbedtls
-- Configuring done (2.4s)
-- Generating done (0.4s)
-- Build files have been written to: /Users/krishnavenkatraman/development/robots/arduino-pico/tools/libpico/build-rp2040
+ make -j
[  0%] Creating directories for 'pioasmBuild'
[  0%] Building ASM object pico-sdk/src/rp2040/boot_stage2/CMakeFiles/bs2_default.dir/compile_time_choice.S.o
[  0%] No download step for 'pioasmBuild'
[  0%] Linking ASM executable bs2_default.elf
[  1%] No update step for 'pioasmBuild'
[  1%] No patch step for 'pioasmBuild'
[  1%] Performing configure step for 'pioasmBuild'
Not searching for unused variables given on the command line.
loading initial cache file /Users/krishnavenkatraman/development/robots/arduino-pico/tools/libpico/build-rp2040/pico-sdk/src/rp2_common/pico_cyw43_driver/pioasm/tmp/pioasmBuild-cache-RelWithDebInfo.cmake
[  1%] Built target bs2_default
[  1%] Generating bs2_default.bin
[  1%] Generating bs2_default_padded_checksummed.S
-- The CXX compiler identification is AppleClang 16.0.0.16000026
[  1%] Building ASM object pico-sdk/src/rp2040/boot_stage2/CMakeFiles/bs2_default_library.dir/bs2_default_padded_checksummed.S.o
-- Detecting CXX compiler ABI info
[  1%] Built target bs2_default_library
[  1%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_adc/adc.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2040/pico_platform/platform.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/hardware_claim/claim.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_sync/sync.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_platform_panic/panic.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_sync_spin_lock/sync_spin_lock.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_sync/sem.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_irq/irq.c.o
[  2%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_irq/irq_handler_chain.S.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_gpio/gpio.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_sync/lock_core.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_sync/critical_section.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_sync/mutex.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_time/time.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_time/timeout_helper.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_util/datetime.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_timer/timer.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_clocks/clocks.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_util/queue.c.o
[  2%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/common/pico_util/pheap.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_pll/pll.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_vreg/vreg.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_bootrom/bootrom.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_ticks/ticks.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_watchdog/watchdog.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_bootrom/bootrom_lock.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_flash/flash.c.o
[  3%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_divider/divider.S.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_boot_lock/boot_lock.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_xosc/xosc.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_dma/dma.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_flash/flash.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_exception/exception.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_xip_cache/xip_cache.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_i2c/i2c.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_pio/pio.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_interp/interp.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_spi/spi.c.o
[  3%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_rtc/rtc.c.o
[  3%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_bit_ops/bit_ops_aeabi.S.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_aon_timer/aon_timer.c.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/hardware_uart/uart.c.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_bootsel_via_double_reset/pico_bootsel_via_double_reset.c.o
[  4%] Building CXX object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_cxx_options/new_delete.cpp.o
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
[  4%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_divider/divider_hardware.S.o
[  4%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_double/double_aeabi_rp2040.S.o
-- Configuring done (0.9s)
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_double/double_init_rom_rp2040.c.o
[  4%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_double/double_v1_rom_shim_rp2040.S.o
[  4%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_float/float_aeabi_rp2040.S.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_double/double_math.c.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_float/float_init_rom_rp2040.c.o
[  4%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_float/float_v1_rom_shim_rp2040.S.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_float/float_math.c.o
[  4%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_int64_ops/pico_int64_ops_aeabi.S.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_malloc/malloc.c.o
[  4%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_mem_ops/mem_ops_aeabi.S.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_multicore/multicore.c.o
[  4%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_unique_id/unique_id.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_runtime/runtime.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_runtime_init/runtime_init_clocks.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_rand/rand.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_runtime_init/runtime_init.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_runtime_init/runtime_init_stack_guard.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_atomic/atomic.c.o
-- Generating done (0.2s)
-- Build files have been written to: /Users/krishnavenkatraman/development/robots/arduino-pico/tools/libpico/build-rp2040/pioasm
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_standard_binary_info/standard_binary_info.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_printf/printf.c.o
[  5%] Building ASM object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_crt0/crt0.S.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_stdlib/stdlib.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_clib_interface/newlib_interface.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_stdio/stdio.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_stdio_uart/stdio_uart.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040/dcd_rp2040.c.o
[  5%] Performing build step for 'pioasmBuild'
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040/rp2040_usb.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/device/usbd_control.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/device/usbd.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/audio/audio_device.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/cdc/cdc_device.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/dfu/dfu_device.c.o
[  5%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/dfu/dfu_rt_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/hid/hid_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/midi/midi_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/msc/msc_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/net/ecm_rndis_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/video/video_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/vendor/vendor_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/net/ncm_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/tusb.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/common/tusb_fifo.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/lib/tinyusb/src/class/usbtmc/usbtmc_device.c.o
[  6%] Building C object CMakeFiles/pico-rp2040.dir/Users/krishnavenkatraman/development/robots/arduino-pico/pico-sdk/src/rp2_common/pico_fix/rp2040_usb_device_enumeration/rp2040_usb_device_enumeration.c.o
[  6%] Linking CXX static library libpico-rp2040.a
ar: stdio.c.obj: not found in archive
ar: stdio_uart.c.obj: not found in archive
ar: stdio_usb.c.obj: not found in archive
ar: stdio_usb_descriptors.c.obj: not found in archive
ar: pico_malloc.c.obj: not found in archive
ar: newlib_interface.c.obj: not found in archive
make[2]: *** [libpico-rp2040.a] Error 1
make[2]: *** Deleting file `libpico-rp2040.a'
make[1]: *** [CMakeFiles/pico-rp2040.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[100%] Built target pioasm
[  6%] Performing install step for 'pioasmBuild'
[100%] Built target pioasm
Install the project...
-- Install configuration: "Release"
[  6%] Completed 'pioasmBuild'
[  6%] Built target pioasmBuild
make: *** [all] Error 2


```

--- a/tools/libpico/CMakeLists.txt
+++ b/tools/libpico/CMakeLists.txt
@@ -312,8 +312,8 @@ target_link_libraries(ipv4-ipv6-bt-big-${cpu}
 
 foreach(tgt pico ipv4 ipv4-ipv6 ipv4-big ipv4-ipv6-big ipv4-bt ipv4-ipv6-bt ipv4-bt-big ipv4-ipv6-bt-big)
     add_custom_command(TARGET ${tgt}-${cpu} POST_BUILD
-        COMMAND ar d lib${tgt}-${cpu}.a stdio.c.obj stdio_uart.c.obj stdio_usb.c.obj stdio_usb_descriptors.c.obj pico_malloc.c.obj newlib_interface.c.obj
-        COMMAND ar d lib${tgt}-${cpu}.a btstack_flash_bank.c.obj  # Need to override with our own implementation
+        COMMAND arm-none-eabi-ar d lib${tgt}-${cpu}.a stdio.c.obj stdio_uart.c.obj stdio_usb.c.obj stdio_usb_descriptors.c.obj pico_malloc.c.obj newlib_interface.c.obj
+        COMMAND arm-none-eabi-ar d lib${tgt}-${cpu}.a btstack_flash_bank.c.obj  # Need to override with our own implementation
         COMMAND cp lib${tgt}-${cpu}.a ../../../lib/${cpu}/lib${tgt}.a
     )
 endforeach()

